ci-operator/step-registry/upi/install/gcp/upi-install-gcp-commands.sh

```diff
@@ -334,13 +334,20 @@ EOF
 
 gcloud deployment-manager deployments create "${INFRA_ID}-bootstrap" --config 04_bootstrap.yaml
 
+INSTANCE_GROUP_SUFFIX="instance-group"
 ## Add the bootstrap instance to the load balancers
 echo "$(date -u --rfc-3339=seconds) - Adding the bootstrap instance to the load balancers..."
 if [ -f 02_lb_int.py ]; then # for workflow using internal load balancers
   # https://github.com/openshift/installer/pull/3270
   # https://github.com/openshift/installer/pull/3309
-  gcloud compute instance-groups unmanaged add-instances "${INFRA_ID}-bootstrap-instance-group" "--zone=${ZONE_0}" "--instances=${INFRA_ID}-bootstrap"
-  gcloud compute backend-services add-backend "${INFRA_ID}-api-internal-backend-service" "--region=${REGION}" "--instance-group=${INFRA_ID}-bootstrap-instance-group" "--instance-group-zone=${ZONE_0}"
+  if gcloud compute instance-groups unmanaged add-instances "${INFRA_ID}-bootstrap-${INSTANCE_GROUP_SUFFIX}" "--zone=${ZONE_0}" "--instances=${INFRA_ID}-bootstrap" 
```

Comment:
I'm not following this conditional. I think the logic we want is: if version >= 4.9 use shortened form, else use old one. 

Is this command some how telling us the version? If so, can you explain how in a comment?

Reply:
Since the installer creates the instance groups with the ig suffix rather than the older -instance-group one, I was hoping to run this command to create instances in the group with the older name. If that does not exist then change the suffix and try again. I am not checking the version but just checking if there is an instance group name with the old suffix first.

Reply:
This makes sense. Thanks for explaining.

Another option you could do is use `gcloud compute instance-groups unmanaged list` with `--filter` to find the existing instance group. That might be cleaner or more explicit, but I also don't see a problem with this approach. Curious what @jstuever thinks?